components/picture_selector.scss

```diff
@@ -0,0 +1,54 @@
+.PictureSelector {
+    display: flex;
+    flex-direction: column;
+    width: 96px;
+}
+
+.PictureSelector__imageContainer {
+    height: 96px;
+    position: relative;
+}
+
+.PictureSelector__image {
+    height: 100%;
+    border-radius: 100%;
+    background-position: 50% 50%;
+    background-size: cover;
+    border: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
+}
+
+.PictureSelector__selectButton {
+    position: absolute;
+    top: 0;
+    right: 0;
+    border-radius: 100%;
+    background-color: var(--center-channel-bg);
+    border: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
+    box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.12);
+    padding: 4px;
+    width: 28px;
+    height: 28px;
+    display: flex;
+    justify-content: center;
+
+    i {
```

Comment:
can we make this more specific with a class? We might end up battling it later with another icon. 🤷‍♀️

Reply:
This is already wrapped around the `PictureSelector__selectButton` class, meaning it only affects icons in that scope, and I can't see us using this class and an icon within it for something else.

Reply:
HTML elements do have more specificity and it can still clash if we are using a sibling component. I have had this issue multiple times when trying to add a new element to the RHS for example. I think we should strive to target specificity only with classes.

Reply:
In general, yes. But I don't think we need to be so specific with this case. There's not going to be any sibling components of `PictureSelector__selectButton`, it's pretty self-contained and I don't see any need to change to allow consumers to add their own select button.